Sources/Datadog/DatadogCore/DatadogCore.swift

```diff
@@ -43,4 +45,14 @@ extension DatadogCore: DatadogCoreProtocol {
         // no-op
         return nil
     }
+
+    // MARK: V1 interface
+
+    func registerFeature(named featureName: String, instance: Any?) {
+        v1Features[featureName] = instance
+    }
+
+    func feature<T>(_ type: T.Type, named featureName: String) -> T? {
+        return v1Features[featureName] as? T
+    }
```

Comment:
Just making sure - this interface is tailored for V1 features, right? Asking, because what we need for V2 is very different 🙂. In V2 features must be instantiated from `registerFeature(named:...)` and we'll only have a single return type in `feature(named:)`.

Reply:
Exactly, I've kept the v2 methods above the `// MARK: V1 interface`. These 2 methods are here while we migrate v1 features, so we can start relying on core for feature registry but we don't have yet storage and upload mechanism in core.